lib/miq_pglogical.rb

```diff
@@ -0,0 +1,87 @@
+class MiqPglogical
+  REPLICATION_SET_NAME = 'miq'.freeze
+  SETTINGS_PATH = [:replication].freeze
+  NODE_PREFIX = "region_".freeze
+
+  def initialize(connection = ApplicationRecord.connection)
+    @connection = connection
+  end
+
+  # Configures the current database server to match the miq server configuration
+  def configure
+    mode = MiqServer.my_server.get_config.config.fetch_path(*SETTINGS_PATH, :mode)
```

Comment:
Can we inject the config, instead of filling this class with calls to MiqServer.my_server.etc?